src/app/shared/table/nested-totals-table.component.ts

```diff
@@ -56,21 +56,11 @@ export class NestedTotalsTableComponent {
   @Input() nestedData: TotalsTableRow[];
   @Input() nestedDataLoading: boolean;
   @Input() nestedDataLoaded: boolean;
-  @Input() nestedDataError: any;
+  @Input() nestedDataErrorActions: ACTIONS.AllActions[];
   @Input() routerParams: RouterParams;
   @Output() discloseNestedData = new EventEmitter<string>();
   getGroupName = getGroupName;
 
-  get nestedDataRetryAction() {
```

Comment:
Looks like this moved into `selectNested500ErrorReloadActions` in the error selector? Following the chain of assignment to get there took me a little while.

Reply:
Yes! Sorry! I factored this out. I was thinking about adding a retry button in the map component as well, but it didn't actually feel right when I did it.